moveit_ros/planning/moveit_cpp/src/planning_component.cpp

```diff
@@ -81,12 +81,6 @@ PlanningComponent::PlanningComponent(const std::string& group_name, const rclcpp
   planning_pipeline_names_ = moveit_cpp_->getPlanningPipelineNames(group_name);
 }
 
-PlanningComponent::~PlanningComponent()
-{
-  RCLCPP_INFO(LOGGER, "Deleting PlanningComponent '%s'", group_name_.c_str());
-  clearContents();
```

Comment:
Would you mind removing the clearContents() function as well?

Reply:
Done, pushed that change. Calling that function could break all sorts of things in this class.

Reply:
Done, pushed that change. Calling that function could break all sorts of things in this class.

Reply:
Done, pushed that change. Calling that function could break all sorts of things in this class.